src/e2e-test/java/io/cdap/plugin/salesforcebatchsource/actions/SalesforcePropertiesPageActions.java

```diff
@@ -0,0 +1,141 @@
+package io.cdap.plugin.salesforcebatchsource.actions;
+
+import io.cdap.e2e.pages.locators.CdfPipelineRunLocators;
+import io.cdap.e2e.utils.SeleniumHelper;
+import io.cdap.plugin.salesforcebatchsource.locators.SalesforcePropertiesPage;
+import io.cdap.plugin.salesforcebatchsource.stepsdesign.DesignTime;
+import io.cdap.plugin.salesforcebatchsource.utils.*;
+import io.cucumber.datatable.DataTable;
+import org.apache.commons.lang3.RandomStringUtils;
+import org.apache.log4j.Logger;
+import org.openqa.selenium.WebElement;
+
+import java.io.IOException;
+import java.util.List;
+import java.util.Map;
+
+import static io.cdap.plugin.salesforcebatchsource.utils.PluginUtils.getPluginPropertyValue;
+
+public class SalesforcePropertiesPageActions {
+    private static final Logger logger = Logger.getLogger(DesignTime.class);
+    public static CdfPipelineRunLocators cdfPipelineRunLocators =
+            SeleniumHelper.getPropertiesLocators(CdfPipelineRunLocators.class);
+
+    static {
+        SeleniumHelper.getPropertiesLocators(SalesforcePropertiesPage.class);
+    }
+
+    public static void selectDataPipelineTypeAsBatch() {
+        WaitHelper.waitForElementToBeClickable(SalesforcePropertiesPage.dataPipelineTypeSelector);
+        logger.info("Click on Data Pipeline dropdown");
+        SalesforcePropertiesPage.dataPipelineTypeSelector.click();
+
+        logger.info("Select Data Pipeline - Batch option");
```

Comment:
We need to include these logs in reports. So, write beforeActions and add this entry.